NEWS.md

```diff
@@ -30,6 +30,80 @@ The format is based on [Keep a Changelog]
 | 2021-04-15 | [0.2.0]       | 0.13.0          | 5011          |
 | 2021-03-05 | [0.1.0]       | 0.12.0          | 4627          |
 
+## [unreleased]
+
+The main focus in this release has been to port labwc to wlroots 0.18 and to
+grind out associated regressions. Nonetheless, it contains a few non-related
+additions and fixes as described below.
+
+As a one-off note, there appears to be an issue with increased commit failures
+when using wlroots-0.18, particularly with intel drivers. If this turns out to
+be an issue for anyone please try running with `WLR_DRM_NO_ATOMIC=1` or revert
+to the v0.17 branch until this is resolved.
```

Comment:
`or run the labwc v0.7 branch or its latest release until this is resolved` ?

Reply:
Done